### ISSUE TITLE
Fix `Goto()` application documentation

### ISSUE THREAD

**Issue by 7-zete-7** [asterisk/documentation#92]
On documentation syntax is `Goto([context,[extensions,]]priority)`, but in real syntax is `Goto([[context,]extension,]priority)` (see https://github.com/asterisk/asterisk/blob/4280d7cd77eb2344db26e51d7bf011e4af10cafd/main/pbx.c#L8841C51-L8841C83).

**Comment by gtjoseph** [comment]
Yeah, I think Gosub is also broken.  I'm looking at the build scripts to see where the issue might be.
.
